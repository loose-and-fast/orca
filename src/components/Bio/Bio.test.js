import React from 'react'

import Bio from './Bio'

describe('Bio', () => {
  let component, props

  beforeEach(() => {
    props = {}
    component = shallow(<Bio {...props} />)
  })

  it('should', () => {
    expect(component).toMatchSnapshot()
  })
})