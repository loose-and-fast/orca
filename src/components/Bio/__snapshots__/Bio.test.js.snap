// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bio should 1`] = `undefined`;
